api/DeviceLightEvent.json

```diff
@@ -34,7 +34,7 @@
             {
               "version_added": "22",
               "version_removed": "61",
-              "notes": "Not supported for macbook with touchbar and <a href='https://bugzil.la/754199'>Windows 7</a>."
+              "notes": "Not supported for MacBook with Touchbar and Windows 7 (see <a href='https://bugzil.la/754199'>bug 754199</a>)."
```

Comment:
Since we're cleaning up this note, I think Apple uses "Touch Bar" (with a space)

Reply:
Fixed in 66b81343ed410a569a22d8c1f310533608272f89.